docs/source/transaction_family_specifications/integerkey_transaction_family.rst

```diff
@@ -0,0 +1,133 @@
+***************************************************
+IntegerKey Transaction Family Specification
+***************************************************
+
+Overview
+=========
+The IntegerKey transaction family allows users to set, increment, and decrement
+the value of entries stored in a state dictionary.
+
+An IntegerKey family transaction request is defined by the following values:
+
+* A verb which describes what action the transaction takes 
+* A name of the entry that is to be set or changed
+* A value by which the entry will be set or changed
+
+The 'set' verb is used to create new entries. The initial value of the entry will be 
+set to the value specified in the transaction request. The 'inc' and 'dec' verbs are 
+used to change the value of existing entries in the state dictionary.
+
+State
+=====
+This section describes in detail how IntegerKey transaction information is stored 
+and addressed.
+
+The address values of IntegerKey transaction family entries are stored in state as
+a CBOR encoded dictionary, with the key being the *Name* and the value being an integer
+*Value*.
+
+.. code-block:: python
+
+    cbor.dumps({
+        // The name of the entry : The current value of the entry
+        'Name':'Value'
+    })
+
+\ *Names* and *Values* must conform to the following rules:
+
+* Valid *Names* are utf-8 encoded strings with a maximum of 20 characters
+* Valid *Values* must be integers in the range of 0 through 2\ :sup:`32` - 1 (32-bit unsigned int)
+
+Addressing
+----------
+IntegerKey data is stored in the state dictionary using addresses which are generated from the IntegerKey namespace prefix and the unique name of the IntegerKey entry. These addresses are generated using the following algorithm:
+
+.. code-block:: pycon
+
+    >>> hashlib.sha512('intkey'.encode('utf-8')).hexdigest()[0:6] + hashlib.sha512('name'.encode('utf-8')).hexdigest()[-64:]
```

Comment:
Should that `[-64:]` be `[64:]`? `hashlib.sha512('name'.encode('utf-8')).hexdigest()` is 128 characters, so it works out the same, but I'm not sure which is intended. It might be helpful to include an English description of the algorithm.